assets/schema_input.json

```
@@ -24,6 +24,9 @@
                 "format": "file-path",
                 "exists": true,
                 "pattern": "^\\S+\\.crai$"
+            },
+            "design": {
+                "type": "string"
```

```suggestion
                "type": "string",
                "meta": "design"
```
I think this makes more sense in the meta map